kubectl-gadget.Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Dockerfile for the kubectl-gadget plugin.
```

Comment:
Maybe add one more sentence regarding the purpose of this container? Given that the plugin is a statically compiled binary, it's not immediately obvious why someone might want to have a container with it.